pkgs/development/compilers/sbcl/2.1.1.nix

```diff
@@ -21,10 +21,6 @@ stdenv.mkDerivation rec {
   buildInputs = [texinfo];
 
   patchPhase = ''
```

Comment:
```suggestion
  postPatch = ''
```

otherwise we can't add patches via the patches variable.

Reply:
Thanks for the suggestion. I've implemented that, and added the missing hooks - the effort's trivial, it reduces complaints from the bot, and the reasoning looks sound.
It still compiles and runs, and the resulting compiler cleanly builds a non-trivial project, so it certainly doesn't seem to have hurt anything.